mariamihaelafainarea/assignment3/Assignment3Test

```diff
@@ -0,0 +1,226 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment3;
+
+import java.util.Iterator;
+import java.util.Set;
+import java.util.TreeSet;
+import jdk.nashorn.internal.ir.Assignment;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author maria
+ */
+public class Assignment3Test {
+    @Test
+    public void simple(){
+        //2x3 matrix !
+        //simple example
+        Assignment3 b = new Assignment3();
+        Assignment3.MyDictionary dictionary =  b.new MyDictionary();
+        dictionary.insert("card");
+        dictionary.insert("cat");
+        dictionary.insert("cart");
+        dictionary.insert("car");
+        char[][] grid = new char[2][3];
```

Comment:
This would be much more readable if you replaced it with
char[][] grid = {{'a','a','r'},{'t', 'c', 'd'}};